WordPress/gutenberg
# Add `Ignore` option to sticky options for Query Loop block

## What?
Query block sticky posts needs "Ignore" option

Closes #66221

## Why?

Currently there are options:

Include: default behavior where sticky posts are prepended to returned posts
Exclude: all sticky posts are excluded
Only: only sticky posts are included

There should be a final option:

Ignore: sticky posts are included in their natural order (basically as if they were not sticky)

This would simply set 'ignore_sticky_posts' => true in the query.

This is very useful for building a "Recent posts" list where we just want most recent regardless of sticky or not.

## How?

It adds an `Ignore` option for user to select and then sets the query var to ignore sticky posts.

## Testing Instructions




1. Create a new post, or set existing post as Sticky
2. Edit a page or template
3. Insert a Query Loop block
4. Uncheck *Inherit query from template*
5. For *Sticky Posts*, select *Ignore* option
6. See sticky posts in natural sorting order

### Testing Instructions for Keyboard


## Screenshots or screencast
